File: low-level/exe7.c
#include <stdio.h>

int main(void) {
    float num, den;
    int t;
    
    printf("Digite quantas vezes deseja relizar a operacao: ");
    scanf("%d", &t);

    for (int i = 0; i < t; i++) {
        printf("\nDigite o numerador: ");
        scanf("%f", &num);
        printf("Digite o valor do denominador: ");
        scanf("%f", &den);
        printf("Resultado: %.2f\n", (num/den));
    }
    return 0;
}

File: logic-exercises/eclipse/exe7.java
package eclipse;

import java.util.Scanner;

public class exe7 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        float num, den;
        int t;

        System.out.println("Digite a quantidade de vezes que deseja realizar a operação: ");
        t = scanner.nextInt();

        for (int i = 0; i < t; i++) {
            System.out.println("Digite o numerador: ");
            num = scanner.nextFloat();
            System.out.printf("Digite o denominador: ");
            den = scanner.nextFloat();
            System.out.println("Resultado: " + num/den);
        }
        scanner.close();
    }
}
